src/pool/poolClient.ts

```diff
@@ -144,7 +147,10 @@ export class PoolEventState {
   private iface: ethers.utils.Interface;
   // maintain ordered unique list of events so we can calculate state
   private events: uma.SerializableEvent[] = [];
-  constructor(private contract: hubPool.Instance, private startBlock = 0) {
+  constructor(
+    private contract: hubPool.Instance,
+    private startBlock = 0
+  ) {
```

Comment:
This rule seems unnecessarily strict to me - isn't it fine to have the constructor arguments on a single line as long as they don't exceed the line length limit? I'd propose to tweak the eslint config for this, rather than change the code.

Reply:
Resolved offline